doc/tutorial/statistical_inference/supervised_learning.rst

```diff
@@ -468,72 +554,58 @@ creating a decision energy by positioning *kernels* on observations:
 
 .. |svm_kernel_linear| image:: /auto_examples/svm/images/sphx_glr_plot_svm_kernels_001.png
    :target: ../../auto_examples/svm/plot_svm_kernels.html
-   :scale: 65
 
 .. |svm_kernel_poly| image:: /auto_examples/svm/images/sphx_glr_plot_svm_kernels_002.png
    :target: ../../auto_examples/svm/plot_svm_kernels.html
-   :scale: 65
 
-.. rst-class:: centered
-
-  .. list-table::
-
-     *
-
-       - **Linear kernel**
-
-       - **Polynomial kernel**
-
-
-
-     *
-
-       - |svm_kernel_linear|
-
-       - |svm_kernel_poly|
-
-
-
-     *
-
-       - ::
+.. |svm_kernel_rbf| image:: /auto_examples/svm/images/sphx_glr_plot_svm_kernels_003.png
+   :target: ../../auto_examples/svm/plot_svm_kernels.html
 
-            >>> svc = svm.SVC(kernel='linear')
+.. raw :: html
 
-       - ::
+   <div class="sk-doc-div">
+     <div class="sk-doc-div-box">
+      <h4>Linear kernel</h4>
```

Comment:
I'm still fighting with `vertical-align` directive to align all the titles to the top. I'm open to suggestions.... :) Thanks!

Reply:
Fixed in 12c04369c4d1093b5aa2d6d14b3ae0432c10f597